### PR TITLE
Add blog post: summary of first 4 dao cycles

### PR DESCRIPTION
**Pull request by m52go**
I am aware that the site header doesn't work correctly on blog post pages. I'm working on a fix now, and will not merge this post until that is fixed (EDIT: [here is PR with fix](https://github.com/bisq-network/bisq-website/pull/212)).

But in the mean time, a review of this post would be great!

Please pay special attention to lines 41 and 43 about seed nodes to make sure they are accurate.

### CODE REVIEW

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "Bisq DAO: The First 4 Cycles"
+author: Steve Jain
+excerpt: "The Bisq DAO launched 4 months ago, after more than 4 years of development. It has now completed 4 voting cycles. In this post, we provide an update on how it's worked out so far. <br><br>"
+---
+
+<div class='responsive-youtube-container'>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/zjdB5_r3mG8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (erciccione) [inline _posts/2019-08-22-bisq-dao-first-four-cycles.md]: I see many other links in the website use `youtube-nocookie.com`. I cannot find any proper documentation about this service, but seems to filter most of the cookies coming from youtube, so i think it's better to use it.

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  * Calendar dates: 5/17/2019 - 6/17/2019
+* 18 proposals
+  * 2 requests to change parameters (trading fees)
+  * 2 general proposals (discussed below)
+  * 14 compensation requests
+* 164 votes cast
+* 22 870 BSQ issued
+* 10 000 BSQ burned
+
+This cycle started right after a couple key developers stepped back from the project—hence the lower issuance.
```
Comment (erciccione) [inline _posts/2019-08-22-bisq-dao-first-four-cycles.md:76]: maybe better to a space before and after `—`

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+## Looking Forward
+
+In my opinion, the launch of the Bisq DAO has been surprisingly smooth.
+
+* BSQ adoption and usage is strong
+* Contributors have been able to sell nontrivial amounts of BSQ for BTC
+* BSQ market has retained strength in the wake of fee increases, which are needed to make the network sustainable
+  * More BSQ was burned in Cycle 4 than was issued
+
+With a functioning revenue model, the network's focus must now be to draw more developers to implement much-needed improvements to the software. [Our call yesterday](https://www.youtube.com/watch?v=NJKldxdEnEY) attracted over 20 potential contributors, some of whom identified themselves as developers. Further efforts to attract contributors will continue.
```
Comment (erciccione) [inline _posts/2019-08-22-bisq-dao-first-four-cycles.md:171]: see comment above about `youtube-nocookie.com`

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+  * 17 compensation requests
+* 302 votes cast
+* 22 730 BSQ issued
+* 20 000 BSQ burned
+
+Following the continued strength of BSQ usage and trading, Cycle 3 voting resulted in another fee increase, this time for both BTC and BSQ fees. Full discussion is [here on GitHub](https://github.com/bisq-network/proposals/issues/99), but the basic reasoning was that strong BSQ markets, strong trading volume, and a need for the network to attract good developers made fee increases toward targets appropriate.
+
+The fee schedule this voting cycle approved look like this:
+
+<img src="../../images/blog/fee-increases-cycle-3.png" alt="Fee increase for Cycle 3`">
```
Comment (erciccione) [inline _posts/2019-08-22-bisq-dao-first-four-cycles.md]: remove ` after *3* in *alt=*

**Review by erciccione — COMMENTED** [review]
Only Minor things.

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+Four months ago, after over four years of development, the Bisq DAO launched.
+
+The objectives are ambitious. The Bisq DAO offers a new way to fund free software, where users pay for usage directly to developers who produce the software. Users also govern the software, making proposals and voting to guide the software's strategy and direction. This paradigm for funding and governing the software also absolves the need for a company, which in turn enables the Bisq network to run without a legal entity or jurisdiction.
+
+If those points don't sound significant to you, here's another way to put it: the code base for the Bisq DAO is bigger than that of the Bisq exchange itself.
+
+Although development and testing were thorough, it didn't feel right to shout from the rooftops about the Bisq DAO being a huge success after just 1 voting cycle. It still doesn't, even after 4 voting cycles. The network still needs to finish some key projects to harden itself, and the DAO itself will likely face continual challenges as it grows and matures.
+
+But it's our duty to keep you (the community) informed, so it's about time we provide on update on how things have gone so far.
```
Comment (devinbileck) [inline _posts/2019-08-22-bisq-dao-first-four-cycles.md]:
```suggestion
But it's our duty to keep you (the community) informed, so it's about time we provide an update on how things have gone so far.
```

**Review by devinbileck — APPROVED** [review]
utACK
Very good summary! Content looks good to me. I did not verify the numbers.

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md (hunk 1 above)
Comment (m52go) [reply _posts/2019-08-22-bisq-dao-first-four-cycles.md]: Yes, thanks -- I've made the change here.

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md (hunk 2 above)
Comment (m52go) [reply _posts/2019-08-22-bisq-dao-first-four-cycles.md:76]: I prefer to use no spaces before and after em dashes if the font allows for it (sometimes fonts don't make the dash as long as it should be, which makes it hard to see).

This is probably why I do tend to use spaces when emulating the em dash with double dashes (e.g., see my comment above). 

Seems like a subjective thing:
https://style.mla.org/is-it-wrong-to-type-spaces-before-and-after-a-dash/

File: _posts/2019-08-22-bisq-dao-first-four-cycles.md (hunk 3 above)
Comment (m52go) [reply _posts/2019-08-22-bisq-dao-first-four-cycles.md:171]: The no-cookie thing only seems to work on embedded videos, so using it here won't work.

**Comment by m52go** [comment]
@erciccione @devinbileck thanks for the feedback!

**Pull request merged**